src/aap_eda/tasks/ruleset.py

```diff
@@ -130,75 +139,93 @@ def deactivate_rulesets(
     ActivateRulesets().deactivate(
         instance,
         deployment_type,
+        final_status,
     )
 
 
-# This is job will be scheduled by a scheduler which currently is unavailable
+# Started by the scheduler, executed by the default worker
 def monitor_activations() -> None:
```

Comment:
Do you need to decorate this function to `default` queue?

Reply:
In my test there is no such need.